library/src/main/java/com/shehabic/droppy/DroppyMenuPopup.java

```diff
@@ -152,12 +152,14 @@ public void dismiss(boolean itemSelected) {
 
     protected void dismissPopup(boolean itemSelected)
     {
-        ((ViewGroup) mContentView.getParent()).removeView(mContentView);
-        ((ViewGroup) modalWindow.getParent()).removeView(modalWindow);
+        if (mContentView != null && modalWindow != null) { 
```

Comment:
When we call dismissPopup() when the popup is not displayed it throws NPE because mContentView is null.